lighthouse-core/lib/i18n/aliases.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const aliases = {
```

Comment:
thank you VERY much for pulling out all these aliases. :)

but since they're pulled out, i would expect to not see them in locales.js anymore.   did you edit that file?